be/src/olap/data_dir.cpp

```diff
@@ -582,7 +576,9 @@ OLAPStatus DataDir::_convert_old_tablet() {
         for (auto& rowset_pb : pending_rowsets) {
             string meta_binary;
             rowset_pb.SerializeToString(&meta_binary);
-            status = RowsetMetaManager::save(_meta, rowset_pb.tablet_uid(), rowset_pb.rowset_id() , meta_binary);
+            RowsetId rowset_id;
+            rowset_id.init(rowset_pb.rowset_id_v2());
```

Comment:
if this is old version rwset_pb, this field will be 0. Is there no problem?

Reply:
No. You see this method is _convert_old_tablet. It is a method to convert old be before 0.10 to current be. During convert process, the rowset_id_v2 is set.